caffe2/python/core.py

```diff
@@ -385,11 +384,11 @@ def CreateOperator(
     # Add rectified inputs and outputs
     inputs = _RectifyInputOutput(inputs)
     outputs = _RectifyInputOutput(outputs)
-    operator.input.extend([text_type(i) for i in inputs])
-    operator.output.extend([text_type(o) for o in outputs])
+    operator.input.extend([str(i) for i in inputs])
+    operator.output.extend([str(o) for o in outputs])
```

Comment:
```suggestion
    operator.input.extend(map(str, inputs))
    operator.output.extend(map(str,outputs))
```
while we are here, might as well make it more efficient.

Reply:
Are you aware of the perf implications? I remember `map` used to be slower than list comprehensions in 2.7 days.

Reply:
```
>>> timeit.timeit('[str(x) for x in range(100)]')
9.269361833000005
>>> timeit.timeit('list(map(str, range(100)))')
8.176639707999982
```
Ok, map is faster